app/components/request_list.tsx

```diff
@@ -101,6 +101,7 @@ const RequestList = (props: RequestObj) => {
     <ReactTable
       data={props.requests}
       columns={columns}
+      resizable={false}
```

Comment:
I think there is nothing wrong in keep it `resizable`.

Reply:
+1 for @amit-mb, Displays the complete URL in expanded state is better than resizing the table